pkg/util/merge/merge_test.go

```diff
@@ -239,7 +239,7 @@ func TestMergeContainer(t *testing.T) {
 
 		t.Run("Volume Mounts are overridden", func(t *testing.T) {
 			volumeMounts := mergedContainer.VolumeMounts
-			assert.Len(t, volumeMounts, 2)
+			assert.Len(t, volumeMounts, 3, "volume mounts can have the same name, the uniqness is the combination of name, path and subpath")
```

Comment:
[nit] Should it be "uniqueness", not "uniqness"?

Reply:
Resolving! You fixed it before I could submit the comment 🎉

Reply:
yes! Addressed in this commit https://github.com/mongodb/mongodb-kubernetes-operator/pull/300/commits/b2896a20a2281c621b1927209bfc73ec36d2ea6a